langlib/langlib-test/src/test/resources/test-src/valuelib_test.bal

```diff
@@ -2432,6 +2432,60 @@ function testCloneWithTypeWithAmbiguousUnion() {
     <string>checkpanic err.detail()["message"]);
 }
 
+public type CodingExtension record {|
+    *Element;
+
```

Comment:
Are we following any convention to keep a new line between the type inclusion and other record fields?

Reply:
It is clear to add a space there.